src/mscorlib/shared/System/ReadOnlySpan.Fast.cs

```diff
@@ -149,6 +149,13 @@ internal ReadOnlySpan(ref T ptr, int length)
 #endif
         }
 
+        /// <summary>
+        /// Returns a reference to the 0th element of the Span. If the Span is empty, returns null reference.
+        /// It can be used for pinning and is required to support the use of span within a fixed statement.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public unsafe ref readonly T GetPinnableReference() => ref (_length != 0) ? ref _pointer.Value : ref Unsafe.AsRef<T>(null);
```

Comment:
@jkotas I cherry-picked only PR i need...i don't expect this if PR was merged...what i'm missing?

Reply:
ok